frontend/src/components/Home.js

```diff
@@ -1,9 +1,126 @@
+        activities: [
+            { id: "1", name: "Rock cycle", description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc maximus, nulla ut commodo sagittis, sapien dui mattis dui, non pulvinar lorem felis nec erat", img: "https://bulma.io/images/placeholders/256x256.png" },
+            { id: "2", name: "Rock cycle", description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc maximus, nulla ut commodo sagittis, sapien dui mattis dui, non pulvinar lorem felis nec erat", img: "https://bulma.io/images/placeholders/256x256.png" },
+            { id: "3", name: "Rock cycle", description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc maximus, nulla ut commodo sagittis, sapien dui mattis dui, non pulvinar lorem felis nec erat", img: "https://bulma.io/images/placeholders/256x256.png" },
+        ],
+        gyms: [
+            { id: "1", name: "Octava compania", img: "https://bulma.io/images/placeholders/256x256.png" },
+            { id: "2", name: "Octava compania", img: "https://bulma.io/images/placeholders/256x256.png" },
+        ]
+    }
+}
 
 function Home() {
+
+    const [homeData, setHomeData] = useState({
+        activities: [],
+        gyms: []
+    })
+    useEffect(() => {
+        const fetchData = async () => {
+            const data = fetchDataFromApi()
+            setHomeData(data)
+        }
+        fetchData()
+    }, [])
+    
     return (
-        <div>
-            [Implement landing page here]
+        <div id="landing-content">
+            <div className="container">
+                <div className="columns">
+                    <div className="column">
+                        <h1 className="title is-size-1 has-text-weight-bold" style={{paddingBottom: 100}}>
+                        Entrena con quien <br></br>quieras y donde sea
+                        </h1>
+                        <ul>
+                            <li className="level-left is-size-5">
+                                <span className="icon is-large has-text-green">
+                                    <i className="far fa-hand-point-right fa-2x"></i>
+                                </span>
+                                <span className="subtitle" style={{paddingLeft: 10}}>Con tu amigo, profe o compañero</span>
+                            </li>
+                            <li className="level-left is-size-5">
+                                <span className="icon is-large has-text-green">
+                                    <i className="far fa-hand-point-right fa-2x"></i>
+                                </span>
+                                <span className="subtitle" style={{paddingLeft: 10}}>Enterate de nuevas clases a <br></br> precios increíbles, y hasta <br></br> gratis.</span>
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="column">
+                        <figure className="image is-3by2">
```

Comment:
Now I think about it, this part of the UI will probably come from API as well at some point. Please extract this into its own Component, maybe call it `Recommender` or similar. Follow the same approach as we did for Gym and Activity.